dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/test-applications/contextandCDI/src/com/ibm/ws/jaxrs20/cdi12/fat/contextandCDI/CDIApplication32.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxrs20.cdi12.fat.contextandCDI;
+
+import java.lang.annotation.Annotation;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import javax.enterprise.inject.Instance;
+import javax.enterprise.inject.spi.CDI;
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Application;
+import javax.enterprise.context.SessionScoped;
+import java.io.Serializable;
+
+@SessionScoped@ApplicationPath("contextandCDI32")
```

Comment:
```suggestion
@SessionScoped
@ApplicationPath("contextandCDI32")
```

Does this work?  I would've expected that white space would be required between two annotations...